### PR TITLE
Fix: typos

### PR DESCRIPTION
**Pull request by omahs**
Classic typo fixor 😎

### CODE REVIEW

**Review by peebeejay — APPROVED** [review]
fantastic finds - ty ty

File: docs/api.mdx [hunk 1]
```diff
@@ -184,7 +184,7 @@ This endpoint allows users to query for a list of all addresses that hold any Gi
 
 ### `GET /v1/address/:address/gitpoaps`
 
-This endpoint allows users to query for some address's (either and ETH or ENS address) GitPOAPs. This returns data like:
```
Comment (burz) [inline docs/api.mdx:187]: I think this should stay as is since without the parenthetical, it would read "address's GitPOAPs"

**Review by burz — CHANGES_REQUESTED** [review]
Just one nitpick, but nice finds!

File: docs/api.mdx (hunk 1 above)
Comment (omahs) [reply docs/api.mdx:187]: Ohh, just changed that one!

**Review by burz — APPROVED** [review]

**Comment by burz** [comment]
Thanks!

**Pull request merged**